plugins/woocommerce/includes/admin/class-wc-admin-addons.php

```diff
@@ -79,31 +79,19 @@ public static function render_featured() {
 	 * @return array|WP_Error
 	 */
 	public static function fetch_featured() {
+		$transient_name = 'wc_addons_featured';
+		// Important: WCCOM Extensions API v2.0 is used.
+		$url = 'https://woocommerce.com/wp-json/wccom-extensions/2.0/featured';
```

Comment:
Should we switch to new URL or are there any risks associated with it?
```suggestion
		$url = 'https://woo.com/wp-json/wccom-extensions/2.0/featured';
```

Reply:
I thought I should leave this one as is for now, so as not to break anything.